CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.221.3 (2022-07-06)
+
+## Fixes
+
+* Patch bug fix on `findTestModule()` inside `index.js` . The fix is compatible to latest Mocha@10.0.0 npm package.
```

Comment:
This was not a bug, it just wasn't compatible with a newer version yet, so how about just:

`testModule: true is now compatible with mocha 10.x`

Reply:
Thank you for copywriting suggestion. I shall change it to yours. 👍🏻